modules/ibi-extra-manifests-configmap.adoc

```diff
@@ -83,6 +83,11 @@ $ oc create configmap sr-iov-extra-manifest --from-file=sriov-extra-manifest.yam
 ----
 configmap/sr-iov-extra-manifest created
 ----
++
+[NOTE]
+====
+If you add more than one extra manifest, and the manifests must be applied in a specific order, you must prefix the filenames of the manifests with numbers representing the order in which they should be applied. For example, `00-namespace.yaml`, `01-sriov-extra-manifest.yaml`, and so on.
+====
 
```

Comment:
@eranco74 - I have a note here specifying the extra manifest order for IBIO as described in @trewest's [bug](https://issues.redhat.com/browse/TELCODOCS-2215).
Would this same note apply to non-IBIO [flow](https://docs.openshift.com/container-platform/4.18/edge_computing/image_base_install/ibi_deploying_sno_clusters/ibi-edge-image-based-install-standalone.html#ibi-extra-manifest-standalone_ibi-edge-image-based-install) also? And what about a similar [section](https://docs.openshift.com/container-platform/4.18/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.html#cnf-image-based-upgrade-prep-extramanifests_cnf-non-gitops) in IBU.

Thanks for your review!

Reply:
The note apply to both IBI, IBIO and IBU